examples/nodejs/aws/lambda-examples/advanced-compression-cdk/src/lambda/index.ts

```diff
@@ -0,0 +1,47 @@
+import {Context, APIGatewayEvent, APIGatewayProxyResult} from 'aws-lambda';
+import {
+  CacheClient,
+  Configurations,
+  MomentoLoggerFactory,
+  DefaultMomentoLoggerFactory,
+  DefaultMomentoLoggerLevel,
+  CompressionLevel,
+  EnvMomentoTokenProvider,
+} from '@gomomento/sdk';
+import {CompressorFactory} from '@gomomento/sdk-nodejs-compression-zstd';
+
+const cacheName = 'cache';
+const cacheKey = 'key';
+const cacheValue = 'value';
+
+const credentialsProvider = new EnvMomentoTokenProvider({
+  environmentVariableName: 'MOMENTO_API_KEY',
+});
+
+const loggerFactory: MomentoLoggerFactory = new DefaultMomentoLoggerFactory(DefaultMomentoLoggerLevel.INFO);
+
+const config = Configurations.InRegion.Default.latest(loggerFactory).withCompressionStrategy({
+  compressorFactory: CompressorFactory.zstd(),
+  compressionLevel: CompressionLevel.SmallestSize,
+});
+
+const defaultTtl = 60;
+const momento = new CacheClient({
+  configuration: config,
+  credentialProvider: credentialsProvider,
+  defaultTtlSeconds: defaultTtl,
+});
+
+export const handler = async (event: APIGatewayEvent, context: Context): Promise<APIGatewayProxyResult> => {
```

Comment:
Can we go ahead and create the cache here too, please? I got a `cache not found` error when I ran the lambda but found that the name of the cache I was supposed to use wasn't documented.